ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -1564,7 +1564,7 @@ class vk_perf_logger {
                 total_op_times += time;
             }
             std::cerr << t.first << ": " << t.second.size() << " x " << (total_op_times / t.second.size() / 1000.0)
-                      << " us";
+                      << " us = " << (total_op_times / 1000.0) << " us";
```

Comment:
One of these units is wrong.

Reply:
The units are both microseconds, the first one is average time and the new one is total time